website/src/blogs/2017-05-23-multi-node-kubernetes-1-6-cluster-provisioning-made-easy-using-sandbox-vagrant-box.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Multi-Node Kubernetes 1.6 Cluster provisioning made easy using SandBox (Vagrant box)
+author: Kiran Mova
+author_info: Contributor and Maintainer OpenEBS projects. Chief Architect MayaData. Kiran leads overall architecture & is responsible for architecting, solution design & customer adoption of OpenEBS.
+tags: Kubeadm, Kubernetes, OpenEBS, Vagrant, Virtualbox
+date: 23-05-2017
+excerpt: Working on OpenEBS, a containerized storage for containers which is orchestrated by Kubernetes, most of our tasks, be it development, testing and demo require us to setup and modify the nodes in Kubernetes cluster.
+---
+
+### Background
+
+Working on OpenEBS, a containerized storage for containers which is orchestrated by Kubernetes, most of our tasks, be it development, testing and demo require us to setup and modify the nodes in Kubernetes cluster. In addition, a multi-node cluster is a must as we go beyond the initial development and testing, to explore the high availability, scale, performance, and upgrade aspects.
+
+While *minikube *and *minishift* provide an easy way to setup Kubernetes single node cluster — for multi-node cluster the fastest ways to get going are usually cloud or hosted solutions. ***kubeadm*** is the closest we can get to easily setup a cluster, but since it is still in alpha, we keep running into some issues like — [kubadm init v1.6.1 fails](https://github.com/kubernetes/kubeadm/issues/226)
```

Comment:
```suggestion
While *minikube* and *minishift* provide an easy way to setup Kubernetes single node cluster — for multi-node cluster the fastest ways to get going are usually cloud or hosted solutions. ***kubeadm*** is the closest we can get to easily setup a cluster, but since it is still in alpha, we keep running into some issues like — [kubadm init v1.6.1 fails](https://github.com/kubernetes/kubeadm/issues/226)
```

Reply:
for font style `italic` type, please use `_some text_`

Reply:
Single star gives italic styling